README.rst

```diff
@@ -436,6 +434,7 @@ The Lightning-Tensor can then be installed with ``pip``:
 
 .. installation_LTensor-end-inclusion-marker-do-not-remove
 
+Lightning-Tensor also requires additional NVIDIA libraries including ``nvJitLink``, ``cuSOLVER``, ``cuSPARSE``, ``cuBLAS``, and ``CUDA runtime``. These can be installed through the `CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit/>`_ or from ``pip``.
```

Comment:
```suggestion
Lightning-Tensor also requires additional NVIDIA libraries including ``nvJitLink``, ``cuSOLVER``, ``cuSPARSE``, ``cuBLAS``, and ``CUDA runtime``. These can be installed through the `CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit/>`_ or from ``pip``. Please ``pip`` install the ``*-cu12`` NVIDIA libraries.
```

Reply:
@maliasadi

Reply:
It would confuses users! Let's not be too specific! They would know which cu* to install depending on their cuda version

Reply:
Let's remove cusolver stuff here